AppController/djinn.rb

```diff
@@ -4950,6 +4950,29 @@ def setup_appengine_application(app, state)
       end
     end
 
+    if my_node.is_login?
+      loop {
+        begin
+          success = uac.add_instance(app, my_public, nginx_port, https_port)
```

Comment:
Would it make sense to move this `uac.add_instance` call to `add_routing_for_appserver`?